granary/meetup.py

```diff
@@ -0,0 +1,102 @@
+# coding=utf-8
+"""Meetup.com source class.
+"""
+
+from . import source
+import logging
+from oauth_dropins.webutil import util
+import urllib.parse, urllib.request
+
+API_BASE = 'https://api.meetup.com'
+API_RSVPS = '/%(urlname)s/events/%(event_id)s/rsvps'
+
+class Meetup(source.Source):
+
+    DOMAIN = 'meetup.com'
+    NAME = 'Meetup.com'
+
+    def __init__(self, access_token):
+        self.access_token = access_token
+        pass
+
+    def create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_formatting=False):
+        return self._create(obj, preview, include_link, ignore_formatting)
+
+    def post_rsvp(self, urlname, event_id, response):
+        url = API_BASE + API_RSVPS % {
+                'urlname': urlname,
+                'event_id': event_id,
+                }
+        params = 'response=%(response)s' % {
+                'response': response,
+                }
+        logging.debug('Creating RSVP=%(rsvp)s for %(urlname)s %(event_id)s' % {
+            'rsvp': response,
+            'urlname': urlname,
+            'event_id': event_id,
+            })
+        return util.urlopen(urllib.request.Request(
+            url,
+            data=params,
+            headers={
+                'Authorization': 'Bearer %(access_token)s' % {'access_token': self.access_token},
+                }
+            ))
+
+    def _create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_formatting=False):
```

Comment:
no preview kwarg here. this and preview_create() should match the method signatures in Source.

Reply:
Sorry my bad! Removed now